src/components/DryRunner/index.js

```diff
@@ -40,6 +41,9 @@ const DryRunner = props => (
             </Checkbox>
           </Column>
         </Row>
+        <Row>
+          <CardLog cards={props.playedCards}></CardLog>
```

Comment:
```suggestion
          <CardLog cards={props.playedCards} />
```

If you do not use `props.children` in your component, you can use a self-closing tag.